var/spack/repos/builtin/packages/py-lightning-uq-box/package.py

```diff
@@ -9,15 +9,18 @@ class PyLightningUqBox(PythonPackage):
     """Lighning-UQ-Box: A toolbox for uncertainty quantification in deep learning."""
 
     homepage = "https://github.com/lightning-uq-box/lightning-uq-box"
-    pypi = "lightning-uq-box/lightning-uq-box-0.1.0.tar.gz"
+    pypi = "lightning-uq-box/lightning_uq_box-0.1.0.tar.gz"
```

Comment:
Doesn't this also have to be 0.2.0?

Reply:
Nope, the version is automatically detected and updated based on which version the user requests